apps/api/infra/api.ts

```diff
@@ -181,6 +181,22 @@ export const serviceSetup = (services: {
       },
       HSN_WEB_FORM_ID: '1dimJFHLFYtnhoYEA3JxRK',
       SESSIONS_API_URL: ref((h) => `http://${h.svc(services.sessionsApi)}`),
+      AUTH_ADMIN_API_PATHS: {
+        dev: json({
+          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
+        }),
+        staging: json({
+          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
+          staging:
+            'https://services-auth-admin-api.internal.staging01.devland.is',
+        }),
+        prod: json({
+          dev: 'https://services-auth-admin-api.internal.dev01.devland.is',
+          staging:
+            'https://services-auth-admin-api.internal.staging01.devland.is',
+          prod: 'https://services-auth-admin-api.internal.innskra.island.is',
+        }),
+      },
```

Comment:
@eirikurn using JSON instead?
Only a bit awkward defining `dev` and `staging` path multiple times 🤔